docs/pipeline/security/customer-managed-keys/index.md

```diff
@@ -0,0 +1,260 @@
+        "kms:GenerateDataKey*",
+        "kms:DescribeKey"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "Allow attachment of persistent resources",
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "arn:aws:iam::SNOWPLOW_PIPELINE_ACCOUNT_ID:root"
+      },
+      "Action": [
+        "kms:CreateGrant",
+        "kms:ListGrants",
+        "kms:RevokeGrant"
+      ],
+      "Resource": "*",
+      "Condition": {
+        "Bool": {
+          "kms:GrantIsForAWSResource": "true"
+        }
+      }
+    }
+  ]
+}
+```
+
+Replace the placeholder values in this policy:
+
+|          Placeholder           |          Replace With          |       Example        |
+| :----------------------------: | :----------------------------: | :------------------: |
+|       `YOUR_ACCOUNT_ID`        |      Your AWS account ID       |    `123456789012`    |
+|       `YourSecurityRole`       | Your security team's role name |  `SecurityTeamRole`  |
+|        `YourAdminRole`         | Your administrative role name  |     `AdminRole`      |
+| `SNOWPLOW_PIPELINE_ACCOUNT_ID` |  Snowplow pipeline account ID  | Provided by Snowplow |
+
+If your security policies don't allow root access (`arn:aws:iam::<account_id>:root`), you can use these specific role patterns instead:
+
+```json
+{
+  "Sid": "Allow use of the key - Specific Roles",
+  "Effect": "Allow",
+  "Principal": {
+    "AWS": [
+      "arn:aws:iam::SNOWPLOW_PIPELINE_ACCOUNT_ID:role/SnowplowOperator",
+      "arn:aws:iam::SNOWPLOW_PIPELINE_ACCOUNT_ID:role/sp-*",
+      "arn:aws:iam::SNOWPLOW_PIPELINE_ACCOUNT_ID:role/snowplow-*"
+    ]
+  },
+  "Action": [
+    "kms:Encrypt",
+    "kms:Decrypt",
+    "kms:ReEncrypt*",
+    "kms:GenerateDataKey*",
+    "kms:DescribeKey",
+    "kms:CreateGrant",
+    "kms:ListGrants",
+    "kms:RevokeGrant"
+  ],
+  "Resource": "*"
+}
+```
+
+Using non-root policies requires coordination with Snowplow support to ensure all necessary roles are included. We strongly recommend the root access approach for simplicity.
+
+## Step 3: Share key information with Snowplow
+
+After creating your key, collect the following information:
+
+- Key ARN: found in the key details page
+  - Format: `arn:aws:kms:REGION:YOUR_ACCOUNT_ID:key/KEY_ID`
+- Key ID: the unique identifier for your key
+- Region: the AWS region where the key was created
+
+Share this information with your Snowplow Customer Success Manager or through a support ticket:
+
+- Key ARN
+- Confirmation that cross-account access is configured
+- Any specific requirements or restrictions
+
+## Step 4: Snowplow configuration
+
+Once you've shared the key information, Snowplow will configure the following pipeline components to use your custom KMS key.
+
+All **pipeline S3 buckets** will be configured with your custom KMS key:
+- Batch archive bucket
+- Batch output bucket
+- Batch processing bucket
+- Kinesis S3 bad events bucket
+- Kinesis S3 enriched events bucket
+- Kinesis S3 raw events bucket
```

Comment:
```suggestion
- S3 bad events bucket
- S3 enriched events bucket
```
Let’s deleted the raw bucket since we are getting rid of it. Also “Kinesis S3” is confusing.

Reply:
Removed raw and removed the Kinesis prefix